reports/README.md

```diff
@@ -16,10 +16,16 @@ make generate_parameters
 make all -j 8
 ```
 
-Finally, to push report data to the production bucket, run the following.
-
-```
-# NOTE: currently need to replace gtfs-data-test with gtfs-data in Makefile
-# to push to production
-make sync
-```
+## Using Postmark to send emails (SAFELY)
+We are currently using the postmark API to send the reports via email to their respective clients
+1. Access to Postmark (contact for access to Bitwarden)
+    There is a Cal-ITP developer shared" collection in Bitwarden that contains keys for the reports sandbox server. An indivudal will not be given access to prod unless it is absolutely necessary. 
+2. Update the config file.
+    all code variables are stored in a config file. config-example.ini is an example config file that shows what the script is expecting in terms of inputs. Create your own config.ini using the example config as a template and populate accordingly. Some commonly updated variables would be month_name or email_subject. If the enviornment is development, you will paste in your sandbox server token in the 'postmark_server_token' spot. Same goes for production. Quotations are not needed for strings. The two different enviornments use two different data inputs. test_emails.csv is a test csv that mimics the production data without any actual client emails. The email_csv_path in production will be the URL from the google doc that will contain client emails. 
+3. Generate the template
+    We are using the mjml markup language to generate the html body and styling of the template. Both the report email and its compiled output is tored in the /templates/email. It is worth mentioning that mjml should use lowercase names to match the config file
+4. Swap out the variables using config file
+    We are using jinja in strict mode to swap out the MJML variables in the template. We use it in strict mode, so if a variable is incorrectly filled in (is misnamed, etc) it will raise an error
```

Comment:
The generation of mjml / population with jinja kind of happens automatically within the script. I'd recommend rewriting step 3 to say that if any changes want to be done to the email contents, then the template should be updated. And then step 4 should be how to execute the email script.